how-to-whitelist-the-elastic-aws-ips-for-telerik-platform.md

```diff
@@ -0,0 +1,47 @@
+---
+title: How to Whitelist the Elastic AWS IPs for Telerik Platform
+description: Information for establishing a secure connection between your machine and Amazon Web Services AppBuilder stays behind.
```

Comment:
You can drop the "information for" part and go with "Establishing a secure connection between your machine and the Amazon Web Services that AppBuilder stays behind."